### DISCUSSION TITLE
Is there anyway to reduce disk/write usage? Maybe an feature request?

### DISCUSSION THREAD

**Discussion by wazerstar** [jef/streetmerchant#1462] [Ideas and feature requests]
loads of terebytes has been written since running streetmerchant for 24/7

I tried using ramdisk it only seem to make it worse.

**Comment by Nephaleem** [comment]
I find it very odd... I have no such problem and I run it pretty much 24/7 too. Only thing I could see is if there was a LOT of alerts, it would pop success proof JPGs in your streetmerchant folder, but man, you'd have to miss a LOT of opportunities in order to run into Terabytes. Where's the data written to, and what files are taking the most space? Very curious...

**Reply by wazerstar** [reply]
Either you are lying or have not clue of what you are talking about and being ignorant.

Here's an video from logging whats going on all the time.

https://streamable.com/dcycuz


Where's your proof of saying you don't have any written data to your disks????

**Reply by tripp396** [reply]
Looking at my data usage, I have this sitting on a 20 GB partition and on Friday the number was around 8GB used, and now it's at 12GB used. So I am seeing 1GB a day used.

**Reply by wazerstar** [reply]
> 
> 
> Looking at my data usage, I have this sitting on a 20 GB partition and on Friday the number was around 8GB used, and now it's at 12GB used. So I am seeing 1GB a day used.

You cannot estimate it like this in general, that would be for your case only, it all depends on how many stores you have configured, the less X stores the less written of course, but I still see this as an big issue since I think the disk write cache can be disabled so its only loaded into memory and cleared when killed again.

**Reply by tripp396** [reply]
That's very true. I did not think about it like that.
